### ISSUE TITLE
ambiguous/missing license

### ISSUE THREAD

**Issue by lucasluitjes** [jrobertson/xinput_wrapper#1]
Hi there, I noticed there is no license in this repository, but on rubygems the gem is marked as MIT. I'd like to re-use the `@lookup` and `@modifiers` tables in my own xinput handling ruby script (and add a little "taken from https://github.com/jrobertson/xinput_wrapper" comment). Can I? Thanks!

**Comment by jrobertson** [comment]
Yes you can use it. I will add an MIT license to the repository when I find the time.

**Comment by lucasluitjes** [comment]
Thanks!
